docker.dev.yml

```diff
@@ -1,10 +1,22 @@
 version: "3"
 
 services:
-  app:
-    container_name: scribble-ocr
+  train:
+    container_name: scribble-ocr-train
     build:
       context: .
+      target: tesseractbuild
```

Comment:
So making the api another stage is great, however you didn't create a stage for training. Instead you have the user dropping directly into the `tesseractbuild` stage. The issue with that is that the pip requirements are not installed yet at this stage and the working directory is set to the app when all the training work really happens out of `/train/tesstrain`